goal_src/jak1/engine/mods/mod-common-functions.gc

```diff
@@ -172,6 +172,25 @@ if the result of rand-vu-int-range is 1, then DANCE! if it is not 1, then Don't
   (none)
   )
 
+(defun spawn-actor-by-name ((name string))
+  ;; Takes in the string of name of a actor, and spawns a new process based on the entity.
+    (let* ((entity-actor (the entity-actor (entity-by-name name)))
+          (type (-> entity-actor etype))
```

Comment:
`type` isn't a keyword I hope? I'd assume it would stop you if so

Reply:
Yeah I just know it works tbh